ambari-server/src/main/resources/common-services/LOGSEARCH/0.5.0/properties/input.config-ambari.json.j2

```diff
@@ -191,18 +191,58 @@
           "map_date":{
             "target_date_pattern":"yyyy-MM-dd HH:mm:ss.SSS"
           }
-
         },
-        "level":{
-          "map_fieldvalue":{
-            "pre_value":"Warning",
-            "post_value":"Warn"
+        "level":[
+          {
+            "map_fieldvalue":{
+              "pre_value":"Severe",
+              "post_value":"ERROR"
+            }
+          },
+          {
+            "map_fieldvalue":{
+              "pre_value":"Warning",
+              "post_value":"WARN"
+            }
+          },
+          {
+            "map_fieldvalue":{
+              "pre_value":"Finer",
+              "post_value":"WARN"
+            }
+          },
+          {
+            "map_fieldvalue":{
+              "pre_value":"Info",
+              "post_value":"INFO"
+            }
+          },
+          {
+            "map_fieldvalue":{
+              "pre_value":"Config",
```

Comment:
I think logfeeder wont recognize this as a log level, as its not in the log level grok pattern

Reply:
probably you can change the grok pattern to DATA from LOGLEVEl and then remap the values as you did

Reply:
I changed the ambari_eclipselink grok pattern to DATA from LOGLEVEL.